src/patternfly/components/Masthead/masthead-logo.hbs

```diff
@@ -1,12 +1,12 @@
 <svg height="36px" viewBox="0 0 679 158" role="img">
   <title>Patternfly logo</title>
   <defs>
-    <linearGradient x1="68%" y1="2.25860997e-13%" x2="32%" y2="100%" id="linearGradient-1">
+    <linearGradient x1="68%" y1="2.25860997e-13%" x2="32%" y2="100%">
```

Comment:
I suspect removing this id has made the logo lose its wings - it looks like that part of the svg is looking for that id.